Add support extended position mode (#98)
youtalk/dynamixel_control: README.md

  <param name="usb_port">/dev/ttyUSB0</param>
  <param name="baud_rate">1000000</param>
  <!-- <param name="use_dummy">true</param> -->

>   <param name="extended_mode">false</param> <!-- Set true if you use the extended position mode of Dynamixel motors -->

Reviewer: <!-- <param name="extended_mode">true</param> -->
```suggestion
  <!-- <param name="extended_mode">true</param> -->
```

</hardware>
```

> Note that the `extended_mode` parameter is optional and set to `false` by default.
> If you use the extended position (multi turn) mode of Dynamixel motors, set it to `true`.
> 

Reviewer: ```suggestion
To use extended position mode on the Dynamixel motor, set the `extended_mode` parameter to `true`.
```
